admin_guide/allocating_node_resources.adoc

```diff
@@ -11,76 +11,69 @@
+reserve are also used by the remaining system components such as *sshd*, *NetworkManager*, and so on. After you specify the resources to reserve, the scheduler has more information about the remaining memory and CPU resources that a node has available for use by pods.
 
 [[allocating-node-settings]]
-== Configuring Nodes for Allocated Resources
+== Configuring nodes for allocated resources
 
-Resources reserved for node components are based on two node settings: `kube-reserved` and `system-reserved`.
+Resources are reserved for node components and system components in {product-title} with the `system-reserved` node setting.
 
-You can use the `kube-reserved` setting to reserve a portion of memory dedicated for kubelet and the `system-reserved` setting for the rest of the system processes. 
+{product-title} does not use the `kube-reserved` setting. Documentation for Kubernetes and some cloud vendors that provide a Kubernetes environment might suggest configuring `kube-reserved`. That information does not apply to a {product-title} cluster.
 
-[NOTE]
-====
-Use caution when setting these parameters. Enforcing `kube-reserved` and `system-reserved` limits
-can lead to critical system services on that node being starved of CPU or OOM killed. For more information
-on the effects of these settings, see xref:computing-allocated-resources[Computing Allocated Resources].
-====
+Use caution when you tune your cluster with resource limits and enforcing limits with evictions. Enforcing `system-reserved` limits can lead to starving critical system services from CPU or killing the critical system services by the OOM killer.
+
+In most cases, tuning resource allocation is performed by making an adjustment and then monitoring the cluster performance with a production-like workload. That process is repeated until the cluster is stable and meets service-level agreements.
+
+For more information on the effects of these settings, see xref:computing-allocated-resources[Computing allocated resources].
 
 [options="header",cols="1,2"]
 |===
 
 |Setting |Description
 
 |`*kube-reserved*`
-| Resources reserved for node resources, such as kubelet, the Docker daemon, SkyDNS, and kube-proxy. Default is none.
+| This setting is not used with {product-title}. Add the CPU and memory resources that you planned to reserve to `*system-reserved*` setting.
 
 |`*system-reserved*`
-| Resources reserved for the remaining system components. Default is none.
+| Resources that are reserved for the node components and system components. Default is none.
 |===
 
-You can see which services are controlled by `system-reserved`  with a tool such as libcgroup:
+You can view the services that are controlled by `system-reserved` with a tool such as libcgroup:
```

Comment:
Usually, we wrap a tool name in backticks: `libcgroup` but perhaps that wasn't the case as much in 3.11. 🤷